WordPress/src/main/res/layout-land/login_signup_screen.xml

```diff
@@ -21,28 +20,39 @@
         app:tabGravity="center"
         app:tabIndicatorHeight="0dp" />
 
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="5dp"
+        android:background="@drawable/login_shadow"
+        android:layout_above="@+id/bottom_buttons"/>
+
     <LinearLayout
```

Comment:
The button shadows are getting clipped by the padding when pressed.  We can add the `android:clipToPadding="false"` attribute to this `LinearLayout` parent view in both portrait and landscape layout files to fix it.  See the screenshots below for illustration.

![clip_portrait](https://user-images.githubusercontent.com/3827611/28378331-09b34df6-6c6d-11e7-9624-b7a945fd4af1.png)

![clip_landscape](https://user-images.githubusercontent.com/3827611/28378335-0c731e72-6c6d-11e7-96c7-134a4a3a06a2.png)